File: dart/lib/BluefinDecryptxP2peError.dart
class BluefinDecryptxP2peError extends Error {
  final bool isBluefinDecryptxP2peError = true;

  final String sdk = 'BluefinDecryptxP2pe';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinDecryptxP2peError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinDecryptxP2peError: ' + code + ': ' + message;
}

File: ts/src/BluefinDecryptxP2peError.ts

import { Context } from './Context'


class BluefinDecryptxP2peError extends Error {

  isBluefinDecryptxP2peError = true

  sdk = 'BluefinDecryptxP2pe'

  code: string
  ctx: Context

  constructor(code: string, msg: string, ctx: Context) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

export {
  BluefinDecryptxP2peError
}

